### PR TITLE
RMG-Py release v3.2.0

### PR DESCRIPTION
**Pull request by sevyharris**
### Motivation or Problem
Branch to eventually create new binaries

### Description of Changes
- [x] update version
- [x] update headers (for 2023)
- [x] write release notes (needs updating for the last 10 months or so)
- [x] rebuild pdf documentation (also needs updating since so much time has passed)
- [x] update conda environment file
- [ ] rebuild conda dependencies that we maintain?
- [x] update release notes with final date of new version
- [x] update version number for conda build in .conda/meta.yaml
- [ ] merge RMG-database PR before this one

Joint PR with RMG-database https://github.com/ReactionMechanismGenerator/RMG-database/pull/590

### CODE REVIEW

**Review by kspieks — CHANGES_REQUESTED** [review]
Thanks for this PR! I skimmed the changes and just wanted to add a few very minor comments.

**Comment by sevyharris** [comment]
> Thanks for this PR! I skimmed the changes and just wanted to add a few very minor comments.

@kspieks thanks for your comments! I pushed the changes, so let me know if I got any of your suggestions wrong. Also, feel free to check out the RMG-Py release notes when I post them (hopefully later today). Any comments will be appreciated :)

File: documentation/source/users/rmg/releaseNotes.rst [hunk 1]
```diff
@@ -77,12 +77,13 @@ Date: July 7, 2022
...
+    - Updated AECs for CCSD(T)-F12/cc-pVXZ-F12 st X = D, T
     - Added frequency factor for b2plypd3/def2tzvp
     - Updated BAC using correct frequency scaling factor for:
-        - CCSD(T)-F12/cc-pVTZ-F12
-        - wb97xd3/def2-tzvp
-        - b97-d3/def2-msvp
+        - wB97X-D3/def2-TZVP
+        - B97-D3/def2-mSVP
+        - CCSD(T)-F12/cc-pVTZ-F12//ωB97X-D3/def2-TZVP
+        - F12/cc-pVDZ-F12//ωB97X-D3/def2-TZVP
```
Comment (kspieks) [inline documentation/source/users/rmg/releaseNotes.rst:143]: Thanks for addressing my minor comments :) Could you double check this line? It seems that the `CCSD(T)-` was accidentally dropped. It should say the same method as the line above: `CCSD(T)-F12/cc-pVDZ-F12//ωB97X-D3/def2-TZVP`. Thanks!

Comment (sevyharris) [reply documentation/source/users/rmg/releaseNotes.rst]: Oof, that was clumsy of me. Should be fixed now!

**Review by rwest — COMMENTED** [review]
How well does b813f458b6e0d211c8f890daac705d73420dcdc5 work and what are the consequences. I like (and in fact advocated for) the motivation - not using rmg channel when we can get away with it - but I now recall why we put some of these pins in: to massively accelerate the conda solving process. Once we realized that the only feasible solution to a consistent set was to get a package from the rmg channel, we put the pin in so that conda found that solution more quickly. Without it, the conda solves were timing out.

**Comment by rwest** [comment]
@mliu49 is https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2311/commits/338504fc40379580d3d2a0a7faf31a38d4e393c4 likely to cause any issues, switching to the conda-forge binary of muq instead of the outdated rmg channel one?  Are the places where muq is used covered in some of our unit, regression, or functional tests?

**Comment by mliu49** [comment]
I can't say for sure whether or not it will be compatible. The main concern would be changes to muq since 2019. The version in the RMG channel is 0.2.0 while conda-forge has builds of 0.3.1-0.4.2.

Although it's probably equally likely that global uncertainty has been broken by changes to RMG, since there aren't any automated tests for the uncertainty module to my knowledge (sorry!). The best options for testing manually would probably be either the [example jupyter notebook](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/ipython/global_uncertainty.ipynb) or the [example for the uncertainty paper](https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fkin.21348&file=kin21348-sup-0002-SuppMat.txt).

If the uncertainty modules work with the old muq package in the RMG channel but not with the newer conda-forge versions, it might be best to reach out to Matthew Parno (who seems to be at Dartmouth now) to see if he can provide advice about what has changed in muq.

**Review by JacksonBurns — APPROVED** [review]
LGTM :tada:!

**Comment by sevyharris** [comment]
v3.2.0 was successfully merged, so closing this

**Pull request closed without merging**